dogfood/tester/disruptions.go

```diff
@@ -0,0 +1,201 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2022 Datadog, Inc.
+
+package main
+
+import (
+	"github.com/DataDog/chaos-controller/api/v1beta1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/intstr"
+)
+
+// Globals
+
+var SELECTOR = []string{"app", "chaos-dogfood-client"}
+var CONTAINER = "client-deploy"
```

Comment:
These can be consts, I think?

Reply:
I dont think you can make string arrays const? but I can make the others.